### DISCUSSION TITLE
Impact-parameter dependent differential cross section

### DISCUSSION THREAD

**Discussion by HoreaF** [MadAnalysis/madanalysis5#89] [Q&A]
Dear MadAnalysis 5 Team

For some time I have been trying to get Impact-parameter dependent differential cross
section (dsigma / db) data for particles produced in heavy ion collisions. I got the lhe file with the total cross section from MadGraph but I don't know how I could get this because I haven't worked with MadAnalysis5 before. Please I need guidance on this issue. Many thanks!!!

Best regards,
Horea

**Comment by jackaraz** [comment]
Hi @HoreaF, have you checked [FAQ for normal mode](http://madanalysis.irmp.ucl.ac.be/wiki/FAQNormalMode)? Details on impact parameter computation can be found in [this paper](https://arxiv.org/abs/2112.05163).
